test/base/src/jvmTest/kotlin/io/realm/kotlin/test/compiler/list/ListTests.kt

```diff
@@ -32,8 +33,8 @@ import kotlin.test.assertTrue
 class ListTests {
 
     private val supportedPrimitiveTypes = TypeDescriptor.classifiers.keys.filter {
-        // Filter out RealmObject
-        it != RealmObject::class
+        // Filter out RealmObject and MutableRealmInt
```

Comment:
Why? It seems like having MutableRealmInts in lists should just work?

Reply:
See other comments.